flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/CatalogTable.java

```diff
@@ -29,4 +32,15 @@
 	 * @return table statistics
 	 */
 	TableStats getStatistics();
+
+	/**
+	 * Check if the table is partitioend or not.
+	 */
+	boolean isPartitioned();
+
+	/**
+	 * Get the partition keys of the table. This will be an empty set if the table is not partitioned.
+	 * @return partition keys of the table.
+	 */
```

Comment:
Document the exception thrown from this method.

Reply:
good catch